client_webhooks.go

```diff
@@ -0,0 +1,23 @@
+package monta
+
+import (
+	"context"
+	"net/url"
+)
+
+// GetWebhookConfigResponse is the response output from the [Client.GetWebhookConfig] method.
+type GetWebhookConfigResponse struct {
```

Comment:
What about just `WebhookConfigResponse` as a name? Or does it not fit into the naming scheme for other parts then?

Reply:
I like the idea but it would get away from the pattern we have with the others. I'll revisit this suggestion after we're set on all webhook endpoints being implemented.